source/draw/Color.ooc

```diff
@@ -133,13 +134,14 @@ ColorRgba: cover {
 	r, g, b, a: Byte
 	normalized ::= FloatTuple4 new(this r as Float / 255, this g as Float / 255, this b as Float / 255, this a as Float / 255)
 	black ::= static This new(0, 0, 0, 255)
+	transparent ::= static This new(0, 0, 0, 0)
```

Comment:
Some tests still use a transparent color for filling the background because it makes sense when representing a draw call that does not cover all pixels.